elephant/signal_proc.py

```diff
@@ -0,0 +1,130 @@
+'''
+signal_proc module
```

Comment:
Yes, true. I am not a fried of long names, but I think it will be good to have explicit module names for a start to avoid confusion, so I would agree with signal_processing.